test/server/cards/01_SOR/units/SteadfastBattalion.spec.js

```diff
@@ -1,48 +1,45 @@
-describe('Steadfast Battalion', function() {
-    integration(function() {
-        describe('Steadfast Battalion\'s ability', function() {
+describe('Steadfast Battalion', function () {
+    integration(function () {
+        describe('Steadfast Battalion\'s ability', function () {
             beforeEach(function () {
                 this.setupTest({
                     phase: 'action',
                     player1: {
-                        base: 'security-complex',
                         groundArena: ['steadfast-battalion', 'battlefield-marine'],
                         leader: { card: 'ig88#ruthless-bounty-hunter', deployed: true }
                     },
-                    player2: {
-                        groundArena: ['frontier-atrt', 'wampa'],
-                        leader: { card: 'boba-fett#daimyo', deployed: true }
-                    }
+                    player2: {}
                 });
             });
 
-            it('should give a unit +2/+2', function () {
+            it('should give a unit +2/+2 if you control a leader unit', function () {
                 this.player1.clickCard(this.steadfastBattalion);
-                this.player1.clickCard(this.p2Base);
                 this.player1.clickCard(this.battlefieldMarine);
                 expect(this.battlefieldMarine.getPower()).toBe(5);
 
                 this.player2.pass();
                 this.player1.clickCard(this.battlefieldMarine);
-                this.player1.clickCard(this.p2Base);
                 expect(this.p2Base.damage).toBe(10);
```

Comment:
Can you do me a small favor and add a comment explaining that it's 10 because of steadfast battalion attack + battlefield marine attack, I'm dumb and was confused for a moment lol (ditto other test)